docker/config/never_on_a_server.env

```diff
@@ -21,7 +21,6 @@ DEBUG=True
 STATIC_ROOT=/tmp/crashstats-static/
 
 # For connecting to local S3, we need to connect insecurely.
-AWS_SECURE=False
```

Comment:
This isn't used in the webapp code.

Reply:
This is set in the settings, but not used in the code, and maybe should be removed as well:

https://github.com/mozilla-services/socorro/blob/c3c44333af0118f60670da10d8709010905c71eb/webapp-django/crashstats/settings/base.py#L567